src/DataSource.ts

```diff
@@ -1,28 +1,10 @@
-import { DataSourceInstanceSettings, MetricFindValue, DataQueryResponse, DataQueryRequest } from '@grafana/data';
-import { DataSourceWithBackend } from '@grafana/runtime';
-import { Observable } from 'rxjs';
-
+import { DataSourceInstanceSettings, MetricFindValue } from '@grafana/data';
+import { DataSourceWithBackend, getTemplateSrv } from '@grafana/runtime';
 import { InfluxQuery, InfluxOptions } from './types';
 
 export class DataSource extends DataSourceWithBackend<InfluxQuery, InfluxOptions> {
-  /** @ngInject */
-  instance: DataSourceInstanceSettings<InfluxOptions>;
-  constructor(instanceSettings: DataSourceInstanceSettings<InfluxOptions>, private templateSrv: any) {
+  constructor(instanceSettings: DataSourceInstanceSettings<InfluxOptions>) {
     super(instanceSettings);
-    this.instance = instanceSettings;
-  }
-
-  query(request: DataQueryRequest<InfluxQuery>): Observable<DataQueryResponse> {
-    request.targets.forEach((target: InfluxQuery, i: number) => {
-      request.targets[i] = {
-        ...target,
-        options: this.instance.jsonData,
-      };
-    });
```

Comment:
@srclosson -- not sure what this was trying to do... should not be necessary

Reply:
Yes, I should revert this...

Reply:
I think I was trying to print out queries, and ended up just removing the console.log...